src/popup/modules/QrCreator.js

```diff
@@ -75,6 +77,27 @@ export function setSize(size) {
     qrCodeLib.set("size", size);
 }
 
+/**
+ * Pre-processes the text before making the QR code. Changes about:reader URLs to regular URLs.
+ * 
+ * @function
+ * @param {string} text 
+ * @returns {string}
+ */
+function preprocess(text) {
+    // check for an about:reader URL
```

Comment:
The comment should be below the if block, should not it? :smile: (otherwise it would have to read "do not check for…"